{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2022-31772",
		"assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
		"assignerShortName": "ibm",
		"requesterUserId": "69938c14-a5a2-41ac-a450-71ed41911136",
		"state": "PUBLISHED",
		"datePublished": "2022-11-11T18:56:12.717Z",
		"dateUpdated": "2024-08-03T07:26:01.046Z",
		"dateReserved": "2022-05-27T15:57:46.681Z"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "9a959283-ebb5-44b6-b705-dcc2bbced522",
				"shortName": "ibm",
				"dateUpdated": "2022-11-11T18:56:12.717Z"
			},
			"title": "IBM MQ denial of service",
			"descriptions": [
				{
					"lang": "en",
					"value": "\nIBM MQ 8.0, 9.0 LTS, 9.1 CD, 9.1 LTS, 9.2 CD, and 9.2 LTS could allow an authenticated and authorized user to cause a denial of service to the MQTT channels. IBM X-Force ID: 228335.\n\n",
					"supportingMedia": [
						{
							"type": "text/html",
							"base64": false,
							"value": "\n\n<span style=\"background-color: rgb(204, 217, 226);\">IBM MQ 8.0, 9.0 LTS, 9.1 CD, 9.1 LTS, 9.2 CD, and 9.2 LTS could allow an authenticated and authorized user to cause a denial of service to the MQTT channels. IBM X-Force ID: 228335.</span>\n\n"
						}
					]
				}
			],
			"affected": [
				{
					"vendor": "IBM",
					"product": "MQ",
					"defaultStatus": "unaffected",
					"versions": [
						{
							"status": "affected",
							"version": "8.0.0, 9.0.0, 9.1.0, 9.2.0, 9.3.0"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "CWE",
							"lang": "en",
							"description": "CWE-20",
							"cweId": "CWE-20"
						}
					]
				}
			],
			"metrics": [
				{
					"format": "CVSS",
					"scenarios": [
						{
							"lang": "en",
							"value": "GENERAL"
						}
					],
					"cvssV3_1": {
						"version": "3.1",
						"vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"attackVector": "NETWORK",
						"attackComplexity": "HIGH",
						"privilegesRequired": "LOW",
						"userInteraction": "NONE",
						"scope": "UNCHANGED",
						"confidentialityImpact": "NONE",
						"integrityImpact": "NONE",
						"availabilityImpact": "HIGH",
						"baseScore": 5.3,
						"baseSeverity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"tags": [
						"vendor-advisory"
					],
					"url": "https://www.ibm.com/support/pages/node/6833806"
				},
				{
					"tags": [
						"vdb-entry"
					],
					"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/228335"
				}
			],
			"source": {
				"discovery": "UNKNOWN"
			},
			"x_generator": {
				"engine": "Vulnogram 0.1.0-dev"
			}
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-03T07:26:01.046Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"vendor-advisory",
							"x_transferred"
						],
						"url": "https://www.ibm.com/support/pages/node/6833806"
					},
					{
						"tags": [
							"vdb-entry",
							"x_transferred"
						],
						"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/228335"
					}
				]
			}
		]
	}
}
